core/src/main/scala/dev/profunktor/fs2rabbit/algebra/Connection.scala

```diff
@@ -47,6 +47,7 @@ object ConnectionResource {
             factory.setPort(firstNode.port)
             factory.setVirtualHost(conf.virtualHost)
             factory.setConnectionTimeout(conf.connectionTimeout)
+            factory.setRequestedHeartbeat(conf.requestedHeartbeat.getOrElse(60))
```

Comment:
⛏️ I'd like to make it clear where we get the defaults from
```suggestion
            factory.setRequestedHeartbeat(conf.requestedHeartbeat.getOrElse(ConnectionFactory.DEFAULT_HEARTBEAT))
```